examples/artemis/artemis_persistence.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: broker.amq.io/v1beta1
 kind: ActiveMQArtemis
 metadata:
-  name: ex-aao
+  name: artemis-broker-tls
 spec:
   deploymentPlan:
     size: 1
```

Comment:
messageMigration requires deploymentPlan.size > 1

Reply:
I left there messageMigration, increased size to 2 and renamed file to be cluster. :-)